pkg/controller/ttlafterfinished/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// TTLAfterFinishedSubsystem - subsystem name used for this controller.
+const TTLAfterFinishedSubsystem = "ttl_after_finished_controller"
+
+var (
+	// timeToDeletionSeconds tracks the time it took to delete the job since it
+	// became eligible for deletion.
+	TimeToDeletionSeconds = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      TTLAfterFinishedSubsystem,
+			Name:           "time_to_deletion_seconds",
```

Comment:
generally durations have the suffix `_duration_seconds`, such as `http_request_duration_seconds`, or `scheduling_duration_seconds`.  This is to distinguish them from timestamps (e.g. `process_start_time_seconds`).

The prefix is up to you, but `time_to_deletion_duration_seconds` sounds a bit odd.  Maybe `deletion_duration_seconds`, or `job_deletion_duration_seconds`?

Reply:
thanks, changed to `job_deletion_duration_seconds`